pandas/tests/series/test_analytics.py

```diff
@@ -336,6 +336,25 @@ def test_describe(self):
                           index=['count', 'unique', 'top', 'freq'])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("tz", [None, "US/Eastern"])
+    def test_describe_dt(self, tz):
+        if tz is None:
```

Comment:
You can put this in `pytest.mark.parametrize` as well.

Reply:
sure